Dynamic Programming/Easy/Dice Combinations/source.cpp

```diff
@@ -0,0 +1,27 @@
+
+  #include<bits/stdc++.h>
+  using namespace std;
+  //
+  #define ll    long long int
+  const ll mod=1e9+7;
+  //
+  //
+    int vec[1000001];
```

Comment:
Can you please remove the unecessary comments `//` in lines 4,7,8 and fix indentation in line 9?

Reply:
Removed // comments and fixed indentation.